rainbow-me/browser-extension
# feat: make sure each route is scrolled to top

Fixes BX-163
Figma link (if any):

## What changed (plus any additional context for devs)
- Watches for route changes and scrolls to the top
- Uses `setTimeout` because I think we need to wait a tick for the page to render? Without the timeout, doesn't work. Let me know if you know why this is happening, or if I'm wrong about this. Was driving me mad.


## Screen recordings / screenshots
https://user-images.githubusercontent.com/3009331/207744566-bb109342-93ac-45e3-8a03-114d167ad38a.mp4




## What to test
- Navigate to settings page
- Scroll
- Click "Translations"
- Navigate back
- Check to see if scrolled to top


## Final checklist
